scripts/run.sh

```diff
@@ -1,34 +1,9 @@
 #!/usr/bin/env bash
 
-command_exists() {
-  command -v "$1" >/dev/null 2>&1
-}
+BEARER_TOKEN=$(curl -s https://hubject.stoplight.io/api/v1/projects/cHJqOjk0NTg5/nodes/6bb8b3bc79c2e-authorization-token | jq -r .data | sed -n '/Bearer/s/^.*Bearer //p')
 
-CSO_OPCP_TOKEN="$1"
-MO_OPCP_TOKEN="$2"
-if [[ "$CSO_OPCP_TOKEN" == "" ]]; then
-  echo "You must provide a bearer token"
-  echo "Usage: run.sh <CSO_OPCP_TOKEN> <MO_OPCP_TOKEN>"
-  echo "       CSO_OPCP_TOKEN and MO_OPCP_TOKEN can be obtained from the Hubject test environment: "
-  echo "       https://hubject.stoplight.io/docs/open-plugncharge/6bb8b3bc79c2e-authorization-token"
-  exit 1
-fi
-CSO_OPCP_TOKEN=${CSO_OPCP_TOKEN#"Bearer "}
-MO_OPCP_TOKEN=${MO_OPCP_TOKEN#"Bearer "}
+# fall back to BEARER_TOKEN if no arg
+CSO_OPCP_TOKEN="${1:-$BEARER_TOKEN}"
+MO_OPCP_TOKEN="${2:-$BEARER_TOKEN}"
 
-shift
-
-# Check if 'docker compose' is available (with space)
-if command_exists "docker compose"; then
-  DOCKER_COMPOSE_CMD="docker compose"
-else
-  # Check if 'docker-compose' is available
-  if command_exists docker-compose; then
-    DOCKER_COMPOSE_CMD="docker-compose"
-  else
-    echo "Error: Neither 'docker-compose' nor 'docker compose' is available. Please install Docker Compose."
-    exit 1
-  fi
-fi
-
-export MO_OPCP_TOKEN=$MO_OPCP_TOKEN; export CSO_OPCP_TOKEN=$CSO_OPCP_TOKEN;$DOCKER_COMPOSE_CMD up "${@:2}"
+export MO_OPCP_TOKEN=$MO_OPCP_TOKEN; export CSO_OPCP_TOKEN=$CSO_OPCP_TOKEN;docker-compose up "${@:2}" --build
```

Comment:
The code to detect whether or not "docker compose" was a command or whether we needed to fall back to the deprecated "docker-compose" command was deliberate. Furthermore, whether or not to perform a build should be optional.

Reply:
Ok, I can revert, just trying to tidy up. I find '--build' a better default when it comes to development, as what inevitably happens, is you make a change and docker compose {down,up} doesn't reflect that change.